### ISSUE TITLE
Can we get a proper documentation for iOS implementation. 

### ISSUE THREAD

**Issue by myubuntuu** [mesibo/messenger-app-ios-swift#6]
Please update the iOS documentation by also how can we customise all UI componant. Please update.

**Comment by beltex** [comment]
Agreed. While the sample projects are a great reference, it's a lot to dig through.

There are no docs for the UI SDK correct?

**Comment by Abishekt97** [comment]
I downloaded and run sample projects. I got errors. Please give the correct sample project for developer reference.
![Screenshot 2020-09-01 at 10 13 20 AM](https://user-images.githubusercontent.com/33828645/91796360-967a9e80-ec3d-11ea-9d99-01a69f7091dc.png)
